sdk/python/kfp/dsl/_volume_op.py

```diff
@@ -66,6 +67,7 @@ def __init__(self,
                  storage_class: str = None,
                  modes: List[str] = None,
                  annotations: Dict[str, str] = None,
+                 generate_name: bool = True,
```

Comment:
This needs to be added as the last argument before **kwargs, otherwise it could break existing code that uses positional arguments.

Reply:
True, I totally missed that. Changed.